content/en/blog/2025/gc-elections.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing the 2025 OpenTelemetry Governance Committee Election
+linkTitle: 2025 GC Election
+date: 2025-09-25
```

Comment:
It looks like the `community` PR has been merged, so let us know when you'd like to merge this post!

```suggestion
date: 2025-09-26
```